WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -121,6 +133,11 @@ class ReaderDiscoverViewModel @Inject constructor(
         }
     }
 
+    private fun onReaderTagClicked(tag: String) {
+        val readerTag = readerUtilsWrapper.getTagFromTagName(tag, INTERESTS)
```

Comment:
I am just putting this comment here to type out the use of the `INTERESTS` here to ensure that I selected the right type.

Reply:
`INTERESTS` tag type is used just to fetch the interests (shown on the interest picker) from the server. They're later saved as `FOLLOWED` tags into the `tbl_tags`. `getTagFromTagName` first looks for a tag in the table, it  won't find any tag with the `INTERESTS` tag type:

https://github.com/wordpress-mobile/WordPress-Android/blob/a3988b1b990862b2e7ee16df6737c449a43349f1/WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderUtils.java#L216-L217

I think right tag type will be 'FOLLOWED' here. 

Having said that, I'll recheck if I can eliminate `INTERESTS` tag type.

Reply:
I believe we'll be using "DEFAULT" type for these. We don't expect anything special from them - we just need to open tag detail, right?

Reply:
Looks like we cannot get a relative endpoint url for DEFAULT type

https://github.com/wordpress-mobile/WordPress-Android/blob/9d3d5af87f60e7008350e308be2eaed863c9a82b/WordPress/src/main/java/org/wordpress/android/ui/reader/services/post/ReaderPostLogic.java#L283-L306

Reply:
Hi friends :) Using `DEFAULT` causes the functionality not to work. As seen below
<kbd><img src="https://user-images.githubusercontent.com/1509205/90077950-09d96200-dcc9-11ea-9425-5afdba95d11b.gif" width="320"></kbd>

![interests_not_visible](https://user-images.githubusercontent.com/1509205/90077950-09d96200-dcc9-11ea-9425-5afdba95d11b.gif)